### ISSUE TITLE
iD editor and internet explorer

### ISSUE THREAD

**Issue by pedrito1414** [hotosm/osm-tasking-manager2#677]
Is there any way of displaying that choosing iD editor in the TM will NOT work with Internet Explorer. Seeing a lot of people give up because they can't open iD and it's not obvious why....

**Comment by pgiraud** [comment]
Is it because iD doesn't work in IE, or is it that Tasking Manager isn't able to open it?

**Comment by pedrito1414** [comment]
Not sure - I don't have IE! My dad phoned as he was trying to get started. I'll find out...

**Comment by bgirardot** [comment]
It is my understanding that IE does not support the iD web editor.

When I try and use iD in IE 11 on Windows 7, OSM will automatically try 
and use Potlatch instead.

**Comment by pedrito1414** [comment]
Yep, just ran a test with my dad. We bypassed the TM and tried to edit in IE and it still reverted to potlatch. So, I think it would be good if the TM tells people that iD won't work with IE, otherwise we'll lose people (and especially older people I would think).

On a side note, new Internet Explorer (called Edge - comes with Windows 10) apparently does open iD...
